Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -28,7 +28,7 @@ There are two categories of libraries:
 
 These libraries can lead to unwanted side-effects:
 
-- A library can contain a vulnerability, which will make the application vulnerable. A good example is `AFNetworking` version 2.5.1, which contained a bug that disabled certificate validation. This vulnerability would allow attackers to execute man-in-the-middle attacks against apps that are using the library to connect to their APIs.
+- A library can contain a vulnerability, which will make the application vulnerable. A good example is `AFNetworking` version 2.5.1, which contained a bug that disabled certificate validation. This vulnerability would allow attackers to execute Machine-in-the-Middle attacks against apps that are using the library to connect to their APIs.
```

Comment:
```suggestion
- A library can contain a vulnerability, which will make the application vulnerable. A good example is `AFNetworking` version 2.5.1, which contained a bug that disabled certificate validation. This vulnerability would allow attackers to execute Machine-in-the-Middle (MITM) attacks against apps that are using the library to connect to their APIs.
```